wiki/Roll-Formulas.md

```diff
@@ -79,201 +71,163 @@
 
 `@attributes.attunement.max` - Maximum number of attunement slots
 
+#### Concentration
+
+`@attributes.concentration.limit` - Maximimum number of effects the actor can sustain concentration on.
+
+`@attributes.concentration.roll.min` and `.max` - The minimum and maximum values of the d20 for concentration-related saving throws.
```

Comment:
These don't do anything currently, do they? Probably shouldn't mention them on the wiki in that case.

Reply:
Agree. Replacing with `.save`, which is a number (added during data prep).

Reply:
`roll.min` and `roll.max` appear in a few other places in this changeset as well. I don't know if perhaps I'm getting confused with the overlap with AEs maybe, but I don't think those fields can even be configured in the UI (yet) so I'm not sure about including them in the wiki at this stage.

Reply:
Just me being focused on conc. They should not be there if they are not functioning yet.